irc/handlers.go

```diff
@@ -81,31 +107,43 @@ func parseCallback(spec string, config *AccountConfig) (callbackNamespace string
 // ACC REGISTER <accountname> [callback_namespace:]<callback> [cred_type] :<credential>
 func accRegisterHandler(server *Server, client *Client, msg ircmsg.IrcMessage, rb *ResponseBuffer) bool {
 	nick := client.Nick()
+
+	if len(msg.Params) < 4 {
+		rb.Add(nil, server.name, ERR_NEEDMOREPARAMS, nick, msg.Command, client.t("Not enough parameters"))
+		return false
+	}
+
+	account := strings.TrimSpace(msg.Params[1])
```

Comment:
`TrimSpace` should be redundant here (but I don't have very strong feelings about removing it)

Reply:
oh um true. fixed, thanks for the pointer